ambry-messageformat/src/main/java/com.github.ambry.messageformat/BlobPropertiesSerDe.java

```diff
@@ -76,4 +98,23 @@ public static void putBlobPropertiesToBuffer(ByteBuffer outputBuffer, BlobProper
     Utils.serializeNullableString(outputBuffer, properties.getOwnerId());
     Utils.serializeNullableString(outputBuffer, properties.getServiceId());
   }
+
+  /**
+   * Serialized {@link BlobProperties} to buffer in version {@link #Version2}
+   * @param outputBuffer the {@link ByteBuffer} to write the {@link BlobProperties}
+   * @param properties the {@link BlobProperties} to be serialized
+   */
+  public static void putBlobPropertiesToBufferV2(ByteBuffer outputBuffer, BlobProperties properties) {
```

Comment:
Yes, I agree with your point. Ideally 2nd method is for tests for now. But in next Pr(when we enable write path), this method will replace the 1st one. Hence I felt its reasonable to leave it there.